components/x-topic-search/src/TopicSearch.scss

```diff
@@ -42,6 +42,11 @@
 	&::placeholder {
 		color: oColorsGetPaletteColor('white');
 	}
+
+	&::-webkit-search-cancel-button {
+	  -webkit-appearance: none;
+		@include oIconsGetIcon('cross', oColorsGetPaletteColor('white'), 26);
```

Comment:
Looks like the indentation isn't quite right here?

Also, I believe it's SASS best practice to `@include` mixins before property declarations. This is just in case the mixin sets the same property you're also trying to set directly - CSS will always take the second value of a duplicate property.

Reply:
@dan-searle ` -webkit-appearance: none;` need to declare before `@include` to remove the default styles applied by Webkit. I'm going to add a comment about that.

Reply:
The `oIconsGetIcon` mixin doesn't output the `-webkit-appearance` property though? 

https://github.com/Financial-Times/o-icons/blob/master/scss/_mixins.scss#L17

Reply:
!?

Reply:
I'm confused.